src/Material/Textfield.elm

```diff
@@ -319,6 +329,24 @@ view lift model options =
         Just val -> [Html.Attributes.maxlength val]
         Nothing -> []
 
+
+    listeners =
+      Dict.values config.listeners
+
+    textValue =
+      case config.value of
+        Just str ->
+          [ Html.Attributes.value str ]
+        Nothing ->
+          []
+
+    defaultInput =
```

Comment:
I think you do this to make sure the field responds to user input even if the parent is not handling inputs. It's a neat idea, but I don't think it'll work, though: Input sets "is-dirty" which is somehow important. (Floating labels, maybe?)